src/frontend/package.json

```diff
@@ -12,10 +12,15 @@
   "license": "MIT",
   "dependencies": {
     "@blueprintjs/core": "^1.6.0",
+    "@types/blueimp-md5": "^1.1.30",
```

Comment:
You probably want to put in a dependency for blueimp-md5 and sprintf via npm -- also jquery.